src/pages/blog/airship-update-april-2020.md

```diff
@@ -27,18 +27,25 @@ The Airship blog is a great way to keep up with what's going on in the community
 posts introduce the changes from Airship 1.0 to Airship 2.0, highlight new features, and detail the evolution of each
 component. The first six Airship 2.0 Blog posts are already available:
 
-- [*Airship Blog Series 6 - Armada Growing Pains*](
-   https://www.airshipit.org/blog/airship-blog-series-6-armada-growing-pains.html)
-- [*Airship Blog Series 5 - Drydock and Its Relationship to Cluster API*](
-  https://www.airshipit.org/blog/airship-blog-series-5-drydock-and-its-relationship-to-cluster-api.html)
-- [*Airship Blog Series 4 - Shipyard - an Evolution of the Front Door*](
-   https://www.airshipit.org/blog/airship-blog-series-4-shipyard-an-evolution-of-the-front-door.html)
-- [*Airship Blog Series 3 - Airship 2.0 Architecture High Level*](
-   https://www.airshipit.org/blog/airship-blog-series-3-airship-2.0-architecture-high-level.html)
-- [*Airship Blog Series 2 - An Educated Evolution*](
-   https://www.airshipit.org/blog/airship-blog-series-2-an-educated-evolution.html)
-- [*Airship Blog Series 1 - Evolution Towards 2.0*](
-  https://www.airshipit.org/blog/airship-blog-series-1-evolution-towards-2.0.html)
+_**UPDATE 03-August-2020:** Airship 2.0 development spans multiple milestones. Upon completing the first major
+milestone, Alpha, the community took some time to reflect on lessons learned and how they impacted the direction and
+design of Airship 2.0. We have summarized these lessons learned and how the design has changed over time - including
+using different technologies and approaches. You can read more about these changes here: [Airship 2.0 is Alpha - Lessons
+Learned](https://www.airshipit.org/blog/airship2-is-alpha/). You should review these changes before reading blogs marked
```

Comment:
I would suggest saying "we recommend reviewing these design changes before reading blogs marked.." so it's a bit less obligatory.

Reply:
Done, in all files.